packages/frontend/hooks/useContractCall.ts

```diff
@@ -1,23 +1,22 @@
-import { useContractCall as useDappContractCall } from '@usedapp/core'
-import { Interface } from '@ethersproject/abi'
+import { useCall as useDappCall } from '@usedapp/core'
 import { useContractConfig } from './useContractConfig'
+import { Contract } from 'ethers'
 
-export function useContractCall<T>(
+export function useCall<T>(
```

Comment:
```suggestion
export function useContractCall<T>(
```

Let's keep the original name to distinguish from the `@usedapp` function and also match the filename.